### ISSUE TITLE
More Minerals

### ISSUE THREAD

**Issue by PowerOfPrime** [petrolpark/Destroy#479]
First,I am Chinese that I am not good at English.Some words may have grammatical and spelling errors.Lots of sentences are machine flipped.I am also a new guy for English-speaking community.
I suggest to add more mineral in Destroy,like blue vitriol,green alum and some other saline mineral.Can use crushing wheel to get corresponding crystalline hydrates powder and use mechanical mixer to mix water and the powder to get corresponding salt solution.The salt block can also as decorative blocks.The solution can get some precious metals by wet metallurgy.
That is my suggestion.

**Comment by FallenNoober** [comment]
agree, perhaps we can refer to the chemical synthesis formulas designed by Gregtech for rapid expansion of elements and synthesis content. 

I really like the chemical reaction mechanism proposed in this mod, and I believe it will be an important component of all integrated packages for mechanical power technology in the future

**Comment by FallenNoober** [comment]
Perhaps it is only necessary to increase the compatibility of chemical elements through tags? 
Using chemical substances provided by other technology mods for chemical reactions is also a convenient method
